### ISSUE TITLE
Better utilize available space for UI on PC

### ISSUE THREAD

**Issue by deee167** [Anuken/Mindustry-Suggestions#21]
Better utilize available space for UI on PC.

It's very inefficient and hard to use the UI's listed below on the PC because everything takes up way too much space (too big font etc) and the available screen space is not used effectively.  Everything is squished into a narrow vertical column that only shows about 4 blocks of info at once (eg. 4 saves per screen).  In reality it should be more like the map selection screen but with options to change to a list view like in windows explorer.  It would be great if the layout of the blocks/cards (eg maps) would have a small thumbnail with a larger area for the map title and description right beside it.  When you clicked on it it would show in full screen the map picture description etc.

Basically everything boils down to this UI being designed for mobile and not PC.  What I suggest is to check the screen resolution or platform and use the space available better and more in line with the standards on that platform.

This is an issue for screens like the following:
map selection
save and load
blueprints
server connection
mods
map editor wave editing
etc.

Thanks

**Comment by deee167** [comment]
Any update on this?

**Comment by Anuken** [comment]
This was implemented to some extent, with exceptions:

> with options to change to a list view like in windows explorer

I'm not interested in maintaining two different types of UI layouts, especially when a list view would rarely be used.

> It would be great if the layout of the blocks/cards (eg maps) would have a small thumbnail with a larger area for the map title and description right beside it

I don't like the idea of smaller map icons. With more columns visible, the current ones are fine.

> too big font

The font size wasn't chosen because of mobile compatibility; I think it looks fine on desktop.

> server connection

I don't see how this could be improved much. Every game I've played with a server list has the servers, well, *in a list*, and the only lists I've seen that are more compact simply display less information per server.

> mods

Same as above. Terraria and Minecraft ['s unofficial mod loaders], for example, both display mods in a narrow list.

> map editor wave editing

Now that the wave editor has a graph in it, adding extra rows isn't viable.
